package.json

```diff
@@ -4,6 +4,7 @@
   "description": "System-wide web3",
   "main": "compiled/main",
   "scripts": {
+    "prepare": "husky install",
```

Comment:
I noticed this in the husky setup docs but does this need to be run every time a new repo is checked out or just once and then we have the husky hooks checked into source control?

Reply:
I'm guessing every time, otherwise it wouldn't exist?